packages/panels/src/Panel/Concerns/HasDefaultThemeMode.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Filament\Panel\Concerns;
+
+trait HasDefaultThemeMode
+{
+    protected string $defaultThemeMode = "system";
+
+    public function defaultThemeMode(string $mode): static
```

Comment:
Please introduce a string-backed enum instead of a string, so we can limit the options people can pass in

Reply:
@danharrin  
it ready to review